jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/JmeterDsl.java

```diff
@@ -188,7 +188,32 @@ public static DslThreadGroup threadGroup(String name) {
    * @since 0.33
    */
   public static DslSetupThreadGroup setupThreadGroup(ThreadGroupChild... children) {
-    return new DslSetupThreadGroup(Arrays.asList(children));
+    return new DslSetupThreadGroup(null, Arrays.asList(children));
+  }
+
+  /**
+   * This allowing to set a name on the setup thread group.
+   * <p>
+   * Setting a proper name allows to properly identify the requests generated in each thread group.
+   *
+   * @see DslSetupThreadGroup
```

Comment:
You might also add a reference to previous method which has more description about setupThreadGroup

Reply:
```Same as {@link #setupThreadGroup(ThreadGroupChild...)}``` and ```@see``` annotations are not enough?